src/led.c

```diff
@@ -20,7 +20,7 @@
 #include <drivers/clock_control.h>
 #include <drivers/gpio.h>
 
-#include "error.h"
+#include "../common/error.h"
```

Comment:
This should be just: #include "common/error.h", there shouldn't be any need to change it.